packages/sdk/src/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { Environment } from './types';
+
+export const DEVNET_FEE_ORACLE_BASE_URL: string = 'https://fee-oracle.develop.buildwithsygma.com/';
```

Comment:
are we going to exposes this?

Reply:
It is a public URL so I don't necessarily see the problem.